client/src/components/eventCard/EventCard.jsx

```diff
@@ -1,34 +1,55 @@
 import styled from 'styled-components';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import edit_icon from '../../assets/edit_icon.png';
 import delete_icon from '../../assets/delete_icon.png';
 import location_icon from '../../assets/location_icon.png';
 import clock_icon from '../../assets/clock_icon.png';
 import calendar_icon from '../../assets/calendar_icon.png';
 import { useState, useEffect } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 const EventCard = ({ showEditButton, eventDetail, deleteEvent }) => {
+  const notify = () => toast("Wow so easy !");
+
   const [confirmedText, setConfirmedText] = useState(' ');
   useEffect(() => {
-    if (eventDetail.votes.length === 1 || eventDetail.votes.length === 0)
-      setConfirmedText('Niemand hat bisher Bock.');
-    else if (eventDetail.votes.length === 2)
-      setConfirmedText(eventDetail.votes[1].name + ' hat Bock.');
-    else if (eventDetail.votes.length === 3)
-      setConfirmedText(eventDetail.votes[1].name + ' und ein anderer hat Bock.');
+    let bockCount = 0;
+
+    eventDetail.votes.forEach((vote) => {
+      if (vote.confirm) {
+        bockCount++;
+      }
+    });
+
+    if (bockCount === 0) setConfirmedText('Niemand hat bisher Bock.');
+    else if (bockCount === 1) setConfirmedText(eventDetail.votes[1].name + ' hat Bock.');
```

Comment:
It was unfortunately not possible for me to see the working version since your deployment seems broken and when I run `npm run dev` the app doesn't come beyond "loading".

However I was wondering why you would want to display the second person in your votes array when the bockCount is 1? Shouldn't it be:
```suggestion
    else if (bockCount === 1) setConfirmedText(eventDetail.votes[0].name + ' hat Bock.');
```
?

Reply:
I've changed it.